core/src/main/scala/com/evolutiongaming/kafka/flow/RebalanceListener.scala

```diff
@@ -5,46 +5,50 @@ import cats.data.NonEmptySet
 import cats.syntax.all._
 import com.evolutiongaming.catshelper.LogOf
 import com.evolutiongaming.kafka.flow.ConsumerFlow.log
-import com.evolutiongaming.kafka.flow.kafka.Consumer
-import com.evolutiongaming.skafka.consumer.{RebalanceListener => SRebalanceListener}
+import com.evolutiongaming.skafka.consumer.RebalanceCallback._
+import com.evolutiongaming.skafka.consumer.RebalanceCallback.syntax._
+import com.evolutiongaming.skafka.consumer.{
+  RebalanceCallback,
+  RebalanceListener1 => SRebalanceListener1,
+  RebalanceListener1WithConsumer
+}
 import com.evolutiongaming.skafka.{Partition, Topic, TopicPartition}
 
 object RebalanceListener {
 
   def apply[F[_]: Monad: LogOf](
-    consumer: Consumer[F],
     flows: Map[Topic, TopicFlow[F]]
-  ): SRebalanceListener[F] = new SRebalanceListener[F] {
+  ): SRebalanceListener1[F] = new RebalanceListener1WithConsumer[F] {
 
-    def onPartitionsAssigned(topicPartitions: NonEmptySet[TopicPartition]) =
+    override def onPartitionsAssigned(topicPartitions: NonEmptySet[TopicPartition]): RebalanceCallback[F, Unit] =
       groupByTopic(topicPartitions) traverse_ { case (topic, flow, partitions) =>
         for {
-          log <- log[F]
-          _ <- log.prefixed(topic).info(s"$partitions assigned")
+          log <- log[F].lift
```

Comment:
please minimise number of `lift` calls

Reply:
like one before  `consumer.position` and one after

Reply:
done